slick-codegen/src/main/scala/slick/codegen/AbstractSourceCodeGenerator.scala

```diff
@@ -59,39 +59,37 @@ abstract class AbstractSourceCodeGenerator(model: m.Model)
 
     //build and nest tuples if > 22 elements
     def tupleBuilder(values: Seq[String] , prefix: String): String = {
```

Comment:
let's call this something like nestedTuple, to be a bit more precise. And the word Builder doesn't add anything to the value of the name. It's accurate but doesn't provide more information, so let's leave it out.